ui/page/exchange/create_order_page.go

```diff
@@ -266,13 +268,13 @@ func (pg *CreateOrderPage) displayCreateWalletModal(isSourceWallet bool, asset l
 }
 
 func (pg *CreateOrderPage) walletCreationSuccessFunc(isSourceWallet bool, asset libutils.AssetType) {
+	pg.OnNavigatedTo()
```

Comment:
Any reason why we are doing this?

Reply:
yeah creating a wallet from the initial disabled page needs the create order page to be reinitialised